Firestore/src/Admin/V1/Gapic/FirestoreAdminGapicClient.php

```diff
@@ -0,0 +1,970 @@
+<?php
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was generated from the file
+ * https://github.com/google/googleapis/blob/master/google/firestore/admin/v1/firestore_admin.proto
+ * and updates to that file get reflected here through a refresh process.
+ *
+ * @experimental
+ */
+
+namespace Google\Cloud\Firestore\V1\Gapic;
```

Comment:
This namespace is incorrect.

Reply:
@jdpedrie 

Thanks!

> Why are there duplicate copies of the same GAPIC?
That's probably the result of my attempts to fix the namespace issue, and forgot to remove the unnecessary one for test to pass although that one is probably the right one in terms of consistent namespace!

> Additionally, since we're adding a new gapic to a GA library, I think we should have a system smoke test.

I agree. Let's add the system tests.